CLAUDE.md

```
@@ -0,0 +1,67 @@
+```
+
+Docker alternative (no local `dotnet` needed):
+```bash
+./docker_build.sh && ./docker_test_bindings.sh
+```
+
+## Architecture
+
+### Code Generation Pipeline
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)